PyRDF/Spark/__init__.py

```diff
@@ -1,16 +1,10 @@
-import PyRDF
-
-
-def make_spark_dataframe(*args, **kwargs):
+def RDataFrame(*args, **kwargs):
```

Comment:
It's ok to do it like this for now, but when we integrate this into ROOT and we do the injection, we'll recheck how we want to do such injection.